src/dbally/similarity/chroma_store.py

```diff
@@ -0,0 +1,96 @@
+from typing import List, Literal, Optional, Union
+
+import chromadb
+
+from dbally.embedding_client.base import EmbeddingClient
+from dbally.similarity.store import SimilarityStore
+
+
+class ChromadbStore(SimilarityStore):
+    """Class that stores text embeddings using [Chroma](https://docs.trychroma.com/)"""
+
+    def __init__(
+        self,
+        index_name: str,
+        client: chromadb.Client,
```

Comment:
I would rename this to `chroma_client` since there is a name clash between "Chroma client" and the "Embedding client" that almost every store takes as a constructor argument.